.github/workflows/client-app.yml

```
@@ -22,7 +22,7 @@ permissions:
   security-events: write
 
 concurrency:
-  group: release-ci-group
+  group: ${{ (github.event_name == 'pull_request' && github.event.pull_request.number) || github.head_ref }}
```

Need `github.workflow` to distinguish from other CI

I suggest use
```suggestion
  group: ${{ github.workflow }}-${{ github.event_name == 'pull_request' && github.head_ref || github.sha }}
```